src/store/applyMiddleware.js

```diff
@@ -0,0 +1,36 @@
+// Function taken from redux source
+// https://github.com/reactjs/redux/blob/master/src/compose.js
+function compose(...funcs) {
```

Comment:
@tshaddix I take it you mean adding these members to `index.d.ts`? Yeah I'm not too knowledgable here. @davidtom let us know if you'd like Tyler to add to that for ya!

Reply:
Yes, that would be great! My only real exposure to typescript has been reading it, so any help with adding code would be much appreciated.

Reply:
@davidtom Cool, well I'm down to merge this and then work on typescript separately.

Reply:
Great, thank you!

And also thank you all for making this great library available. It's been really instrumental in the chrome extension we've been working on.

Reply:
@davidtom You're welcome! At this point the library is really built by all the awesome contributors such as yourself! Thanks for making it even better!